CHANGES.rst

```diff
@@ -2,10 +2,13 @@
 Changes
 =======
 
-1.1 (unreleased)
+1.0.1 (unreleased)
 ----------------
```

Comment:
ReStructuredText requires the underlines to be same length or longer than the title, otherwise you get a rendering error.

Reply:
I would have use for a ReStructuredText linter for sublime to prevent these small mistakes. Good, that you have spotted it.